fastly/resource_fastly_service_v1.go

```diff
@@ -2653,18 +2660,26 @@ func resourceServiceV1Update(d *schema.ResourceData, meta interface{}) error {
 			return fmt.Errorf("[ERR] Invalid configuration for Fastly Service (%s): %s", d.Id(), msg)
 		}
 
-		log.Printf("[DEBUG] Activating Fastly Service (%s), Version (%v)", d.Id(), latestVersion)
-		_, err = conn.ActivateVersion(&gofastly.ActivateVersionInput{
-			Service: d.Id(),
-			Version: latestVersion,
-		})
-		if err != nil {
-			return fmt.Errorf("[ERR] Error activating version (%d): %s", latestVersion, err)
-		}
+		shouldActivate := d.Get("activate").(bool)
+		if shouldActivate {
+			log.Printf("[DEBUG] Activating Fastly Service (%s), Version (%v)", d.Id(), latestVersion)
+			_, err = conn.ActivateVersion(&gofastly.ActivateVersionInput{
+				Service: d.Id(),
+				Version: latestVersion,
+			})
+			if err != nil {
+				return fmt.Errorf("[ERR] Error activating version (%d): %s", latestVersion, err)
+			}
 
-		// Only if the version is valid and activated do we set the active_version.
-		// This prevents us from getting stuck in cloning an invalid version
-		d.Set("active_version", latestVersion)
+			// Only if the version is valid and activated do we set the active_version.
+			// This prevents us from getting stuck in cloning an invalid version
+			d.Set("active_version", latestVersion)
+		} else {
+			log.Printf("[INFO] Skipping activation of Fastly Service (%s), Version (%v)", d.Id(), latestVersion)
+			log.Print("[INFO] The Terraform definition is explicitly specified to not activate the changes on Fastly")
```

Comment:
This line should be also `log.Printf`?

Reply:
[Printf](https://golang.org/pkg/fmt/#Printf) formats according to a format specifier and writes to standard output. The string: `The Terraform definition is explicitly specified to not activate the changes on Fastly` doesn't need to be formatted. Therefore we just need to use [Print](https://golang.org/pkg/fmt/#Print) in this case in order to write the string to standard output.

Here is an example how the info messages are formatted in standard output:

```
[INFO] Skipping activation of Fastly Service (5XTpVB5M69L4UBQBlX70zE), Version (2)
[INFO] The Terraform definition is explicitly specified to not activate the changes on Fastly
[INFO] Version (2) has been pushed and validated
[INFO] Visit https://manage.fastly.com/configure/services/5XTpVB5M69L4UBQBlX70zE/versions/2 and activate it manually
```